e2e/tests/stepDefinitions/manage/bookingSearch.ts

```diff
@@ -94,7 +94,7 @@ When('I click on the Provisional bookings tab', () => {
 When('I search for a valid CRN in provisional bookings', () => {
   cy.then(function _() {
     const bookingSearchPage = Page.verifyOnPage(BookingSearchPage, 'provisional')
-    bookingSearchPage.searchByCRN(this.booking.person.crn, 'provisional')
+    bookingSearchPage.searchByCrnOrName(this.booking.person.crnOrName, 'provisional')
```

Comment:
Shouldn't this still be `this.booking.person.crn`? The relevant E2E test doesn't seem to pass for me locally